### PR TITLE
007 matrix of roles

### PR DESCRIPTION
**Pull request by KewinTaoAnh**

### CODE REVIEW

File: test/unit/service/BandServiceTests.js [hunk 1]
```diff
@@ -29,4 +30,28 @@ describe('interface',function(){
...
+        })
+
+         it('should throw exception when 500 error returned from axios', async () => {
+                var mock = new MockAdapter(axios);
+
+                mock.onGet(interface.URL).reply(500);
+
+                var error = await interface.getBandLevelNames();
+
+
```
Comment (hellofadude) [inline test/unit/service/BandServiceTests.js:53]: remove extra line

File: test/unit/service/JobServiceTests.js [hunk 2]
```diff
@@ -46,4 +47,26 @@ describe('JobService', function () {
...
+    expect(results[0]).to.deep.equal(role);
+  })
+
+  it('should throw exception when 500 error returned from axios', async () => {
+    var mock = new MockAdapter(axios);
+    mock.onGet(interface.URL).reply(500);
+
+    try{
+      var error = await interface.getJobRolesByCapability(-1);
+    }catch(e){
```
Comment (hellofadude) [inline test/unit/service/JobServiceTests.js:67]: space out this line properly to improve readability

**Review by hellofadude — APPROVED** [review]
Approved. Minor comments which you can self resolve when done

**Review by JennicaMistry — APPROVED** [review]

File: test/unit/service/JobServiceTests.js (hunk 2 above)
Comment (KewinTaoAnh) [reply test/unit/service/JobServiceTests.js:75]: done

File: test/unit/service/BandServiceTests.js (hunk 1 above)
Comment (KewinTaoAnh) [reply test/unit/service/BandServiceTests.js]: done

**Pull request merged**